### ISSUE TITLE
Why I got the different score result from your pretrained model？

### ISSUE THREAD

**Issue by Wini1680** [eurecom-asp/RawGAT-ST-antispoofing#1]
this is my env

Package            Version

appdirs            1.4.4
audioread          2.1.9
certifi            2021.10.8
cffi               1.15.0
charset-normalizer 2.0.7
decorator          5.1.0
idna               3.3
joblib             1.0.1
librosa            0.8.0
llvmlite           0.37.0
mkl-fft            1.3.1
mkl-random         1.2.2
mkl-service        2.4.0
numba              0.54.1
numpy              1.19.2
olefile            0.46
packaging          21.2
Pillow             8.4.0
pip                21.2.4
pooch              1.5.2
protobuf           3.19.1
pycparser          2.21
pyparsing          2.4.7
PyYAML             5.4.1
requests           2.26.0
resampy            0.2.2
scikit-learn       1.0.1
scipy              1.7.2
setuptools         58.0.4
six                1.16.0
SoundFile          0.10.3.post1
tensorboardX       2.2
threadpoolctl      3.0.0
torch              1.10.0
torchaudio         0.10.0
torchvision        0.11.1
typing-extensions  3.10.0.2
urllib3            1.26.7
wheel              0.37.0

CUDA Version is 11.2
the result just show the tail result of RawGAT_ST_mul Scores as examples:

tail Pre_trained_models/RawGAT_ST_mul/Scores.txt
LA_E_1774414 A19 spoof -6.717050552368164
LA_E_6774439 A08 spoof -8.20712947845459
LA_E_2104346 A10 spoof -9.96189022064209
LA_E_3635712 A10 spoof -6.151601314544678
LA_E_4078898 A10 spoof -6.435843467712402
LA_E_1665632 - bonafide 6.490197658538818
LA_E_5085671 A09 spoof -10.491961479187012
LA_E_4926022 A16 spoof -3.727647066116333
LA_E_2894498 - bonafide 5.0631890296936035
LA_E_4689563 A19 spoof -3.8826699256896973

and my result is :

LA_E_1774414 A19 spoof -0.1255655288696289
LA_E_6774439 A08 spoof -0.009253472089767456
LA_E_2104346 A10 spoof -0.13266447186470032
LA_E_3635712 A10 spoof -0.15753477811813354
LA_E_4078898 A10 spoof -0.16004103422164917
LA_E_1665632 - bonafide -0.13468778133392334
LA_E_5085671 A09 spoof -0.14242157340049744
LA_E_4926022 A16 spoof -0.17268329858779907
LA_E_2894498 - bonafide -0.14125970005989075
LA_E_4689563 A19 spoof -0.15430286526679993

so why there is different of between you provide Scores.txt and I?
